### PR TITLE
Extract church categories from commons for later use

### PR DESCRIPTION
**Pull request by lokal-profil**
Task: [T176879](https://phabricator.wikimedia.org/T176879)

### CODE REVIEW

**Comment by lokal-profil** [comment]
This was a quick implementation. it might have python 3 compatibility issues and we may want to change e.g. the format of the Municipality names.

File: importer/load_church_cats.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    name = name.partition(',')[0]
+    if any(name.endswith(end) for end in endings):
+        church_dict[name] = church_cat.title()
+
+
+def has_subcats(cat):
+    return len(list(cat.subcategories())) > 0
+
+
+if __name__ == 'main':
```
Comment (Vesihiisi) [inline importer/load_church_cats.py]: `if __name__ == "__main__":`

File: importer/load_church_cats.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+def get_all_church_cats():
+    site = pwb.Site('commons', 'commons')
+    top_cat = pwb.Category(site, 'Category:Churches in Sweden by municipality')
+    church_cats_per_muni = {}
+    for sub_cat in top_cat.subcategories():
+        # municipal level
+        sub_cat_name = sub_cat.title(withNamespace=False)
+        if not sub_cat_name.startswith('Churches in '):
+            raise pwb.Error('doh')
```
Comment (Vesihiisi) [inline importer/load_church_cats.py]: What does this error message mean?

File: importer/load_church_cats.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                (name.endswith(parent_ending) or
+                 name.startswith('Churches in '))):
+            loop_over_candidates(
+                church_cat, church_dict, parent_ending, depth=depth+1)
+        else:
+            add_if_likely_church(church_cat, church_dict)
+
+
+def add_if_likely_church(church_cat, church_dict):
+    endings = ('kyrka', 'kloster', 'kapell', 'kyrkan', 'klostret', 'kapellet')
```
Comment (Vesihiisi) [inline importer/load_church_cats.py]: Add 'kyrkoruin'? Theoretically they should not be in 'Churches in .. municipality' categories, but they often are (https://commons.wikimedia.org/wiki/Category:Karlskyrka_kyrkoruin, https://commons.wikimedia.org/wiki/Category:Kila_kyrkoruin)

Comment (Vesihiisi) [reply importer/load_church_cats.py]: Add 'missionshus', there's a couple directly under 'Churches in ... municipality' categories.

File: importer/load_church_cats.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    name = church_cat.title(withNamespace=False)
+    name = name.partition(',')[0]
+    if any(name.endswith(end) for end in endings):
```
Comment (Vesihiisi) [inline importer/load_church_cats.py]: `name.lower()` in case of misused capitalization.

File: importer/load_church_cats.py (hunk 1 above)
Comment (lokal-profil) [reply importer/load_church_cats.py]: done

File: importer/load_church_cats.py (hunk 2 above)
Comment (lokal-profil) [reply importer/load_church_cats.py]: Done

File: importer/load_church_cats.py (hunk 3 above)
Comment (lokal-profil) [reply importer/load_church_cats.py]: Done

File: importer/load_church_cats.py (hunk 4 above)
Comment (lokal-profil) [reply importer/load_church_cats.py]: Done

**Pull request merged**